src/main/java/cloud/fogbow/ras/core/plugins/interoperability/aws/compute/v2/AwsV2ComputePlugin.java

```diff
@@ -258,19 +232,19 @@ protected ComputeInstance mountComputeInstance(Instance instance, List<Volume> v
 		List<String> privateIpaddresses;
 		String publicIpAddress;
 		for (int i = 0; i < instance.networkInterfaces().size(); i++) {
-			privateIpaddresses = addPrivateIpAddresses(instance, i);
+			privateIpaddresses = retrievePrivateIpAddresses(instance, i);
 			if (!privateIpaddresses.isEmpty()) {
 				ipAddresses.addAll(privateIpaddresses);
 			}
-			publicIpAddress = addPublicIpAddress(instance, i);
+			publicIpAddress = retrievePublicIpAddresses(instance, i);
 			if (publicIpAddress != null) {
 				ipAddresses.add(publicIpAddress);
 			}
 		}
 		return ipAddresses;
 	}
 
-	private String addPublicIpAddress(Instance instance, int index) {
+	private String retrievePublicIpAddresses(Instance instance, int index) {
```

Comment:
this is an index of ...? improve this doc. Also, `retrieve` does not seem a good name. why not get?